docs/restore/rclone.md

```diff
@@ -0,0 +1,108 @@
+# Restore rclone
+
+This will guide you through restoring object storage.
```

Comment:
Very nice that you made a doc for this!
Could you expand this first section with some context on why this could be needed and what this will do? Maybe also include a warning that this will override some data in the "main" buckets.

Reply:
Done :+1: 

Made me realise I didn't document the point in time restore so added that as well.